shareable-dataframes/Dockerfile

```diff
@@ -60,11 +55,13 @@ RUN conda env create -f /rapids/conda/shareable_dataframes.yml --name shareable_
 ADD . /rapids/
 
 # Configure
+## 75->Tuning
+## check if NATIVE should fix it
 RUN source activate shareable_dataframes \
     && cmake \
     -DBUILD_TESTS=OFF \
     -DBUILD_BENCHMARKS=OFF \
-    -DCMAKE_CUDA_ARCHITECTURES= \
```

Comment:
Automatic detection with empty `DCMAKE_CUDA_ARCHITECTURES` was failing.   In my tests, I was getting :
```
Failed to find a working CUDA architecture.
```

Manually fixing seems to do the trick.